cmd/cluster/powervs/create.go

```diff
@@ -0,0 +1,172 @@
+package powervs
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"syscall"
+
+	apifixtures "github.com/openshift/hypershift/api/fixtures"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	powervsinfra "github.com/openshift/hypershift/cmd/infra/powervs"
+	"github.com/openshift/hypershift/cmd/log"
+	"github.com/openshift/hypershift/support/infraid"
+	"github.com/spf13/cobra"
+)
+
+const (
+	defaultCIDRBlock = "10.0.0.0/16"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "powervs",
+		Short:        "Creates basic functional HostedCluster resources on PowerVS PowerVS",
+		SilenceUsage: true,
+	}
+
+	opts.PowerVSPlatform = core.PowerVSPlatformOptions{
+		APIKey:     os.Getenv("IBMCLOUD_API_KEY"),
+		Region:     "us-south",
+		Zone:       "us-south",
+		VpcRegion:  "us-south",
+		SysType:    "s922",
+		ProcType:   "shared",
+		Processors: "0.5",
+		Memory:     32,
+	}
+
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.ResourceGroup, "resource-group", "", "IBM Cloud Resource group")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.Region, "region", opts.PowerVSPlatform.Region, "IBM Cloud PowerVS region")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.Zone, "zone", opts.PowerVSPlatform.Zone, "IBM Cloud PowerVS zone")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.CloudInstanceID, "cloud-instance-id", "", "IBM PowerVS Cloud Instance ID")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.CloudConnection, "cloud-connection", "", "IBM Cloud PowerVS Cloud Connection")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.VpcRegion, "vpc-region", opts.PowerVSPlatform.VpcRegion, "IBM Cloud VPC Region for VPC resources")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.Vpc, "vpc", "", "IBM Cloud VPC Name")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.SysType, "sys-type", opts.PowerVSPlatform.SysType, "System type used to host the instance(e.g: s922, e980, e880)")
```

Comment:
Can we clarify in the text output these are for the default NodePool created by the cli? e.g "System type used to host the machines in the NodePool" (e.g: s922, e980, e880)"

Reply:
Added default values